Building mcap_vendor from source cannot clone

## Description

When building `mcap_vendor` from source with branch `humble`, there are issues checking out the upstream mcap repository.  It appears to be related to the use of `git lfs` in the https://github.com/foxglove/mcap repository.

### Expected Behavior

`mcap_vendor` clones `mcap` and builds/installs it correctly.

### Actual Behavior

The build fails:

```
-- Found ament_cmake: 1.3.3 (/home/mjcarroll/workspaces/ros2_humble/install/share/ament_cmake/cmake)
-- Found zstd_vendor: 0.15.3 (/home/mjcarroll/workspaces/ros2_humble/install/share/zstd_vendor/cmake)
[0/8] Performing download step (git clone) for 'mcap-populate'
Cloning into 'mcap-src'...
Downloading docs/img/0398582-hero.png (669 KB)
Error downloading object: docs/img/0398582-hero.png (266f9d3): Smudge error: Error downloading docs/img/0398582-hero.png (266f9d3dd91c1bd739e1d79ae4abd459f71e8d4c661656745504ea52913467b5): batch request: ERROR: invalid repository: 'foxglove/mcap.git/info/lfs': exit status 1

Errors logged to /home/mjcarroll/workspaces/ros2_humble/build/mcap_vendor/_deps/mcap-src/.git/lfs/logs/20221207T125324.767015936.log
Use `git lfs logs last` to view the log.
error: external filter 'git-lfs filter-process' failed
fatal: docs/img/0398582-hero.png: smudge filter lfs failed
CMake Error at mcap-subbuild/mcap-populate-prefix/tmp/mcap-populate-gitclone.cmake:40 (message):
  Failed to checkout tag: 'dc6561d9ba867901709e36526dcf7f7359861e9c'


FAILED: mcap-populate-prefix/src/mcap-populate-stamp/mcap-populate-download /home/mjcarroll/workspaces/ros2_humble/build/mcap_vendor/_deps/mcap-subbuild/mcap-populate-prefix/src/mcap-populate-stamp/mcap-populate-download 
cd /home/mjcarroll/workspaces/ros2_humble/build/mcap_vendor/_deps && /usr/bin/cmake -P /home/mjcarroll/workspaces/ros2_humble/build/mcap_vendor/_deps/mcap-subbuild/mcap-populate-prefix/tmp/mcap-populate-gitclone.cmake && /usr/bin/cmake -E touch /home/mjcarroll/workspaces/ros2_humble/build/mcap_vendor/_deps/mcap-subbuild/mcap-populate-prefix/src/mcap-populate-stamp/mcap-populate-download
ninja: build stopped: subcommand failed.

CMake Error at /usr/share/cmake-3.22/Modules/FetchContent.cmake:1087 (message):
  Build step for mcap failed: 1
Call Stack (most recent call first):
  /usr/share/cmake-3.22/Modules/FetchContent.cmake:1216:EVAL:2 (__FetchContent_directPopulate)
  /usr/share/cmake-3.22/Modules/FetchContent.cmake:1216 (cmake_language)
  /usr/share/cmake-3.22/Modules/FetchContent.cmake:1259 (FetchContent_Populate)
  CMakeLists.txt:34 (fetchcontent_makeavailable)
  CMakeLists.txt:77 (build_mcap_vendor)


-- Configuring incomplete, errors occurred!
See also "/home/mjcarroll/workspaces/ros2_humble/build/mcap_vendor/CMakeFiles/CMakeOutput.log".
See also "/home/mjcarroll/workspaces/ros2_humble/build/mcap_vendor/CMakeFiles/CMakeError.log".
```

The most relevant bit is:

```
Error downloading object: docs/img/0398582-hero.png (266f9d3): Smudge error: Error downloading docs/img/0398582-hero.png (266f9d3dd91c1bd739e1d79ae4abd459f71e8d4c661656745504ea52913467b5): batch request: ERROR: invalid repository: 'foxglove/mcap.git/info/lfs': exit status 1
```

## To Reproduce

Clone `rosbag2` and build with `colcon build --packages-up-to mcap_vendor`

## System (please complete the following information)
 - OS: Ubuntu Jammy
 - ROS 2 Distro: humble
 - Version: humble (source)

--- a/mcap_vendor/CMakeLists.txt
+++ b/mcap_vendor/CMakeLists.txt
@@ -27,9 +27,8 @@ endif()
 macro(build_mcap_vendor)
   include(FetchContent)
   fetchcontent_declare(mcap
-    GIT_REPOSITORY https://github.com/foxglove/mcap.git
-    GIT_TAG dc6561d9ba867901709e36526dcf7f7359861e9c # releases/cpp/v0.7.0
-  )
+    URL https://github.com/foxglove/mcap/archive/refs/tags/releases/cpp/v0.7.0.tar.gz
+    URL_HASH SHA1=0fccc7bf49e3d8f4dc05219f537cf7d9fa4adb42)
   fetchcontent_makeavailable(mcap)
 
   fetchcontent_declare(lz4

